README.md

```diff
@@ -166,6 +166,12 @@ aws-adfs integrates with:
     password = your_password
     ```
 
+* .aws/credentials profile for automatically refreshing credentials
```

Comment:
There should probably be a warning about the security implications of doing this, refering to the one from AWS docs:
```suggestion
* .aws/credentials profile for automatically refreshing credentials

> Warning: see [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sourcing-external.html) about security considerations to take when sourcing credentials with an external process.
```